eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -7,9 +7,13 @@ parameters:
   _const_paths:
     _wasm_specific_only: [
         eng/testing/ProvisioningVersions.props
+        eng/testing/Was*Runner*
```

Comment:
`(m|i)` instead of `*`?

Reply:
or use `?` to be clear that it's one character

Reply:
with the caveat that i don't know what kind of glob/regex syntax is being used here :-)

Reply:
Fixed. these are used with `git diff`, and follow `git pathspec`.

Reply:
hmph azdo didn't like `?`: `/eng/pipelines/common/evaluate-default-paths.yml (Line: 10, Col: 24): While parsing a flow sequence, did not find expected ',' or ']'.`

Reply:
Took the easy way out with `WasmRunner*`, and `WasiRunner`.